server/controllers/interviewController.js

```diff
@@ -28,4 +30,14 @@ module.exports = {
     const interview = await interviewModel.createInterview(req);
     res.send({ interview });
   },
+  endInterview: async (req, res) => {
+    const id = req.params.id;
+    if (!mongoose.Types.ObjectId.isValid(id))
+      res.status(400).json({ msg: 'Invalid ID' });
+
+    const interviewDoc = await Interview.findById(id);
+    interviewDoc.endTime = Math.floor(Date.now() / 1000);
+    await interviewDoc.save(); // what constitutes a completed interview?
+    res.json(interviewDoc); // Feedback submitted? Review period ended?
```

Comment:
This will be fleshed out more in the front-end - it will be an option for either one of the users to explicitly end the interview. This will help with letting people rejoin if their connection fails.

Reply:
Cool, figured it would be tied into the sockets